checkbox_spec.rb

```diff
@@ -11,24 +11,24 @@
 
   describe "#exists?" do
     it "returns true if the checkbox button exists" do
-      expect(browser.checkbox(:id, "new_user_interests_books")).to exist
-      expect(browser.checkbox(:id, /new_user_interests_books/)).to exist
-      expect(browser.checkbox(:name, "new_user_interests")).to exist
-      expect(browser.checkbox(:name, /new_user_interests/)).to exist
-      expect(browser.checkbox(:value, "books")).to exist
-      expect(browser.checkbox(:value, /books/)).to exist
+      expect(browser.checkbox(id: "new_user_interests_books")).to exist
+      expect(browser.checkbox(id: /new_user_interests_books/)).to exist
+      expect(browser.checkbox(name: "new_user_interests")).to exist
+      expect(browser.checkbox(name: /new_user_interests/)).to exist
+      expect(browser.checkbox(value: "books")).to exist
+      expect(browser.checkbox(value: /books/)).to exist
       # not sure what :text is supposed to represent here
```

Comment:
That's fine; just wanted to make note of it as long as I'm going through all of the code. :)